src/cartridges/int_adyen_SFRA/cartridge/client/default/js/adyen_checkout/renderGenericComponent.js

```diff
@@ -222,25 +225,30 @@ async function initializeCheckout() {
   setAmazonPayConfig(store.checkout.paymentMethodsResponse);
   document.querySelector('#paymentMethodsList').innerHTML = '';
 
-  renderStoredPaymentMethods(
-    store.checkout.paymentMethodsResponse,
-    session.imagePath,
-  );
+  const paymentMethodsWithoutGiftCards =
```

Comment:
Instead of filtering the gift cards, can we also add `giftcard` as blocked type in the configuration?